tests/run-make/naked-symbol-visibility/rmake.rs

```diff
@@ -0,0 +1,98 @@
+//@ ignore-windows
+//@ only-x86_64
```

Comment:
Is the `only-x86_64` really necessary?

Reply:
yes, the assembly will break on aarch64 right?

Reply:
Right